data/transactions/application_test.go

```diff
@@ -33,19 +33,21 @@ import (
 
 func TestApplicationCallFieldsNotChanged(t *testing.T) {
 	partitiontest.PartitionTest(t)
+	t.Parallel()
 
 	af := ApplicationCallTxnFields{}
 	s := reflect.ValueOf(&af).Elem()
 
-	if s.NumField() != 13 {
-		t.Errorf("You added or removed a field from transactions.ApplicationCallTxnFields. " +
-			"Please ensure you have updated the Empty() method and then " +
+	if s.NumField() != 14 {
+		t.Errorf("You  added or removed a field from transactions.ApplicationCallTxnFields. " +
+			"Please ensure you have updated ApplicationCallTxnFields.Empty() and then " +
 			"fix this test")
 	}
 }
 
 func TestApplicationCallFieldsEmpty(t *testing.T) {
 	partitiontest.PartitionTest(t)
+	t.Parallel()
```

Comment:
add `Access` to this test? maybe to rewrite it with reflection and per-field checks to make sure all fields are covered and `Empty()` actually works?

Reply:
The new reflection test ought to cover everything.